pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
 
 [project]
 name = "opensafely-databuilder"
 description = ""
-dynamic = ["version"]
+version = "2+local"
```

Comment:
This will mean pip reports a different version to what's in the version file, was that intended?

Reply:
It wasn't intended. However, I'm unsure how to reconcile dynamic metadata with local version identifiers. Do you know, @ghickman?

Reply:
@iaindillingham – you should be able to use your existing creation of a `__version__` attribute on the package to look it up [dynamically with flit](https://flit.pypa.io/en/latest/pyproject_toml.html#new-style-metadata).  Alternatively you could just use the same string for both 😁